build.sbt

```diff
@@ -43,6 +43,12 @@ lazy val sfsBackend = (project in backendRoot / "sfs")
   .dependsOn(gcsFileSystem)
   .dependsOn(backend % "test->test")
 
+lazy val ecsBackend = (project in backendRoot / "ecs")
```

Comment:
Maybe `amazonBackend` would be more intuitive?  Unless we expect multiple Amazon-flavored backends.